chat_shell/tests/tools/test_mcp_loader.py

```
@@ -0,0 +1,230 @@
+        with (
+            patch(
+                "chat_shell.tools.mcp.loader.settings.CHAT_MCP_SERVERS",
+                json.dumps({"mcpServers": mcp_servers_config}),
+            ),
+            patch(
+                "chat_shell.tools.mcp.MCPClient", return_value=mock_client
+            ) as mock_mcp_class,
+        ):
+            mock_client.connect = MagicMock()
+
+            result = await load_mcp_tools(
```

Ruff (0.15.2)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)